xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/widgets/notification.css

```diff
@@ -2,24 +2,27 @@
 .xnotification-container {
   position : fixed;
   bottom: 0;
-  left: 0%;
-  width: 100%;
+  left: 0;
+  right: 0;
+  margin: auto;
   text-align: center;
   z-index: 1200;
   display: block !important;
+  max-width: 40%;
 }
 .xnotification-wrapper {
   padding-top: 1px;
 }
 .xnotification {
   cursor: pointer;
-  display: inline;
+  display: inline-block;
   background: #333 2px center no-repeat;
   border: none;
-  padding: 0px 6px;
+  padding: 14px 28px;
+  border-radius: 3px;
```

Comment:
Even if we can't use LESS here it would be good to use the values from the skin. We use 4px border radius by default for instance, as for the padding, you should maybe look for some UI element that is similar (uses similar padding, maybe breadcrumbs). My point is to try to have a consistent style across XWiki.

Reply:
I made this more consistent indeed in the latest commit, using the same padding as the one in the breadcrumb, and the default border radius.